app/styles/map.scss

```diff
@@ -54,13 +54,42 @@
 }
 
 #snapmapapp {
-  margin: 0 -15px;
+  position: absolute;
```

Comment:
I made a few changes here to better use the `scss` ability to nest things.  It looks like more could be done in this direction, but it's not really critical right now.